src/reactionsystem.jl

```diff
@@ -1456,7 +1461,7 @@ function Base.convert(::Type{<:JumpSystem}, rs::ReactionSystem; name = nameof(rs
     JumpSystem(eqs, get_iv(flatrs), sts, ps;
                observed = MT.observed(flatrs),
                name,
-               defaults = MT.defaults(flatrs),
+               defaults = _merge(defaults,MT.defaults(flatrs)),
```

Comment:
Does merging after flattening give the correct namespacing of parameters and variables? This seems like it is going to cause problems. Merging probably needs to happen before flattening.

Reply:
Not, that is a good point. I just used the current order of things, but changing it around might be better.

Reply:
Can you add a test to check this?

Reply:
You mean providing defaults to a `convert` for a system with sub-systems, which got defaults?

Reply:
Yes